addons/mail/models/mail_template.py

```diff
@@ -146,17 +145,17 @@ def _search_template_category(self, operator, value):
             ('module', '!=', '__export__')
         ]).subselect('res_id')
 
-        domain = []
+        domains = []
         if 'hidden_template' in value:
-            domain.append(['|', ('active', '=', False), '&', ('description', '=', False), ('id', 'in', templates_with_xmlid)])
+            domains.append(['|', ('active', '=', False), '&', ('description', '=', False), ('id', 'in', templates_with_xmlid)])
 
         if 'base_template' in value:
-            domain.append(['&', ('description', '!=', False), ('id', 'in', templates_with_xmlid)])
+            domains.append([('active', '=', True), ('description', '!=', False), ('id', 'in', templates_with_xmlid)])
```

Comment:
Adding active True?
I guess it's to combine with active False of the first one, but was there a bug or is it a limitation of Domain?

Reply:
It is a small bug. If you have 'active' in the domain, the test_active flag is ignored. All inactive instances have "hidden_template" as view, so the other values must be active.